helper/include/odp/helper/threads.h

```diff
@@ -286,39 +276,6 @@ int odph_thread_create(odph_thread_t thread[],
  */
 int odph_thread_join(odph_thread_t thread[], int num);
 
-/**
- * Creates and launches odpthreads (as linux threads or processes)
- *
- * Creates, pins and launches threads to separate CPU's based on the cpumask.
- *
- * @param thread_tbl    Thread table
- * @param mask          CPU mask
- * @param thr_params    ODP thread parameters
- *
- * @return Number of threads created
- *
- * @deprecated Use odph_thread_create() instead.
- */
-int ODPH_DEPRECATE(odph_odpthreads_create)(
```

Comment:
`odph_odpthreads_create()` is still mentioned in `odph_thread_create()` and `odph_thread_param_t` spec.

Reply:
V2.